maas/plugins/galera_check.py

```diff
@@ -45,10 +47,12 @@ def generate_query(host, port):
         port = ' -P %s' % port
     else:
         port = ''
-
-    return ('/usr/bin/mysql --defaults-file=/root/.my.cnf'
-            '%s%s -e "SHOW STATUS WHERE Variable_name REGEXP '
-            "'^(wsrep.*|queries)'\"") % (host, port)
+    if output_type == 'status':
+        return ('/usr/bin/mysql --defaults-file=/root/.my.cnf '
+                '%s%s -e "SHOW GLOBAL STATUS"') % (host, port)
+    elif output_type == 'variables':
+        return ('/usr/bin/mysql --defaults-file=/root/.my.cnf '
+                '%s%s -e "SHOW VARIABLES"') % (host, port)
```

Comment:
because the variables we grab do not exist as session variables, so the default action is to pass back the global variables anyway.
Having said that, it's probably not a bad idea to force this to global in case we add a metric in the future and erroneously return the session variable which would not be what we want.